pep-0636.rst

```diff
@@ -392,10 +427,6 @@ The simplest form compares a subject value against one or more literals::
                 return "Unauthorized"
             case 403:
                 return "Forbidden"
-            case 404:
-                return "Not found"
-            case 418:
-                return "I'm a teapot"
```

Comment:
I understand the desire to shorten the example -- but 404 is the most well known error code in the world, and 418 brings a bit of levity. I'd drop 401 and 403 instead.